cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -108,22 +108,24 @@ func TestDrain(t *testing.T) {
 		},
 	}
 
-	statefulset := appsv1beta1.StatefulSet{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "ss",
-			Namespace: "default",
-			SelfLink:  "/apiv1s/extensions/v1beta1/namespaces/default/statefulsets/ss",
-		},
-	}
+	/*	Disable stateful set test for a moment due to fake client problems with handling v1beta1 SS
```

Comment:
I think we should create an issue to re-enable it. Otherwise we'll forget about it.